addons/confirm-actions/addon.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "Confirm actions",
-  "description": "Asks if you're sure when committing actions under your Scratch account, such as sharing projects. ",
+  "name": "Confirmations",
+  "description": "Asks if you're sure when committing actions under your Scratch account, such as sharing projects.",
```

Comment:
Not sure about this addon name. Sounds weird. Any other creative ideas?

Reply:
I disagree with apple and say `Action conformation` is fine

Reply:
I think any name for this addon other than "Confirm actions" will sound at least a bit weird.